src/android/build.gradle

```diff
@@ -39,7 +39,7 @@ dependencies {
     implementation 'com.google.firebase:firebase-core:16.0.1'
     implementation 'com.google.firebase:firebase-config:16.0.0'
     implementation ('com.android.support.constraint:constraint-layout:1.0.2')
-    implementation ('com.listonic.ad:companion:2.5.1')
+    implementation ('com.listonic.ad:companion:2.5.1DEV1')
```

Comment:
co to za konstrukcja?

Reply:
dodał mi specjalnie metodę którą mogę sprawdzić czy wszystko ok. na razie chcę ją zostawić póki nie ma nowej wersji od nich